src/main/java/org/mitre/fhir/authorization/AuthorizationController.java

```diff
@@ -214,8 +210,6 @@ private String generateBearerToken(HttpServletRequest request, String clientId,
       throw new ResponseStatusException(HttpStatus.UNAUTHORIZED, "No patients found");
     }
 
-    // get their id
-    // String patientId = patient.getIdElement().getIdPart();
     List<String> scopesList = Arrays.asList(scopes.split(" "));
```

Comment:
This should be split on something like:

```suggestion
    List<String> scopesList = Arrays.asList(scopes.split(" +"));
```

So that it works when multiple spaces are between scopes.

e.g. doing using this scope:

`launch/patient openid fhirUser offline_access patient/Medication.read patient/Condition.read  patient/Observation.read patient/Patient.read` which has 2 spaces between the Condition and Observation scopes

gives:

![Screen Shot 2020-09-08 at 1 46 46 PM](https://user-images.githubusercontent.com/41651655/92510852-7c5f3400-f1da-11ea-8153-5324ce6faea3.png)

Reply:
Also may need to be updated here too as @okeefm pointed out:

https://github.com/inferno-community/inferno-reference-server/pull/17/files#diff-ad5676917be3354a5199f68027f4e7b0R43

Reply:
Still seeing the issue even with the updated commits.

I ran `docker-compose up --build` and made sure to pull the changes.  Should I be doing something else?

Reply:
oh I realize that maybe the regex is still not extracting each of the spaces , like it it being timid. Do you mind if I move this to another small ticket, where I'll handle splitting the scope to and from an array in a separate method?

Reply:
That works.  Can you add a reference to the ticket number here once its created?